mappings/net/minecraft/util/PackedIntegerArray.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS yi net/minecraft/util/PackedIntegerArray
+	FIELD a storage [J
+	FIELD b elementBits I
+	FIELD c maxValue J
```

Comment:
Given this is an integer array, what does this long mean? did mojang just wrote bad code?

Reply:
It's an array of elementBits-bit integers (in the sense of whole numbers, not Java ints) packed into a long array. So for example, if elementBits is 8, each long in storage would contain 4 array elements.